classes/driver/field/abstract.php

```diff
@@ -149,7 +149,7 @@ public function renderErrorValueHtml($value)
      * @param string $defaultValue
      * @return mixed
      */
-    public function getInputValue($defaultValue = '')
+    public function getInputValue($defaultValue = null)
```

Comment:
Let me know if I am mistaken:

sanitazeValue casts 
(string) null
which gives an empty string, the previous $defaultValue ?

Reply:
Yes, but the `sanitizeValue` method may be overriden by some fields that are not strings (like the input number).

Reply:
I agree with @Seb-C, the abstract method should be as neutral as possible, `null` as default value is better.

Could you juste update the PHPDoc according to your changes ? Maybe something like this :

```
@param mixed|null $defaultValue
```